nilearn/datasets/neurovault.py

```diff
@@ -969,15 +971,28 @@ def _get_batch(query, prefix_msg="", timeout=10.0, verbose=3):
     )
     prepped = session.prepare_request(req)
     logger.log(
-        f"{prefix_msg}getting new batch: {query}",
+        f"{prefix_msg}getting new batch:\n\t{query}",
```

Comment:
it is convenient to have most log messages on one line (makes logs easier to grep or parse). for some it's not convenient eg when displayng a traceback, but for this one it might be worth it

Reply:
was just trying to made the output more human readable